Add metrics and event docs (#260)
karmada-io/website: docs/reference/instrumentation/event.md

---
title: Karmada Event Reference
---

## Events

> This page details the event that different Karmada components export.

Reviewer: ```suggestion
This section details the events that different Karmada components export.
```

> You can obtain the event by `kubectl describe` the corresponding object.

Reviewer: ```suggestion
You can obtain query events by running `kubectl describe` on the corresponding object.
```

---
title: Karmada Event Reference
---

## Events

This section details the events which record key processes in Karmada.
See more details about event [here](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/).

| Reason                                  | Binding Objects                                                                             | Type    | Source Component                                                                                                          |
|-----------------------------------------|---------------------------------------------------------------------------------------------|---------|---------------------------------------------------------------------------------------------------------------------------|

> | CreateExecutionSpaceFailed              | cluster                                                                                     | Warning | cluster-controller                                                                                                        |

Reviewer 2: ```suggestion
| CreateExecutionSpaceFailed              | Cluster                                                                                     | Warning | cluster-controller                                                                                                        |
```

| TaintClusterByConditionFailed           | cluster                                                                                     | Warning | cluster-controller                                                                                                        |
| RemoveTargetClusterFailed               | cluster                                                                                     | Warning | cluster-controller                                                                                                        |
| SyncImpersonationConfigFailed           | cluster                                                                                     | Warning | unified-auth-controller                                                                                                   |
| SyncImpersonationConfigSucceed          | cluster                                                                                     | Normal  | unified-auth-controller                                                                                                   |

> | ReflectStatusFailed                     | work                                                                                        | Warning | work-status-controller                                                                                                    |

Reviewer 2: ```suggestion
| ReflectStatusFailed                     | Work                                                                                        | Warning | work-status-controller                                                                                                    |
```

| InterpretHealthFailed                   | work                                                                                        | Warning | work-status-controller                                                                                                    |
| InterpretHealthSucceed                  | work                                                                                        | Normal  | work-status-controller                                                                                                    |
| SyncFailed                              | work                                                                                        | Warning | execution-controller                                                                                                      |
| SyncSucceed                             | work                                                                                        | Normal  | execution-controller                                                                                                      |

> | CleanupWorkFailed                       | resourceBinding<br/>clusterResourceBinding                                                  | Warning | binding-controller<br/>cluster-resource-binding-controller                                                                |

Reviewer 2: ```suggestion
| CleanupWorkFailed                       | ResourceBinding<br/>ClusterResourceBinding                                                  | Warning | binding-controller<br/>cluster-resource-binding-controller                                                                |
```
